libs/ui/editor/src/lib/Atoms/comment/commentPreview.tsx

```diff
@@ -8,10 +8,9 @@ import {useTranslation} from 'react-i18next'
 import {MdAttachFile, MdEdit, MdExpandLess, MdExpandMore, MdTag} from 'react-icons/md'
 import {Link} from 'react-router-dom'
 import {Col, FlexboxGrid, Form, Grid, IconButton, Panel, Row} from 'rsuite'
-import FlexboxGridItem from 'rsuite/esm/FlexboxGrid/FlexboxGridItem'
 
-import {RichTextBlock} from '../../blocks/richTextBlock/richTextBlock'
-import {RichTextBlockValue} from '../../blocks/types'
+import {RichTextBlock} from '../../Blocks/richTextBlock/rich-text-block'
```

Comment:
Can we keep the folder names lowercase? Or why was it changed to be uppercase?

Reply:
I changed them to be uppercase, but yeah, I brought back lowercase not to introduce more confusion

Reply:
I changed them to be uppercase, but yeah, I brought back lowercase not to introduce more confusion